install_config/storage_examples/gluster_dynamic_example.adoc

```diff
@@ -14,272 +15,35 @@ toc::[]
-----
-<1> The defined GlusterFS endpoints. In this example, they are called `heketi-storage-endpoints`.
-
-[NOTE]
-====
-By default, `user_authorization` is disabled. If enabled, you may need to find
-the rest user  and rest user secret key. (This is not applicable for this
-example, as any values will work).
-====
-
-[[create-a-storage-class-for-your-glusterfs-dynamic-provisioner]]
-== Create a Storage Class for Your GlusterFS Dynamic Provisioner
-
-xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[Storage
-classes] manage and enable persistent storage in {product-title}.
-Below is an example of a _Storage class_ requesting 5GB of on-demand
-storage to be used with your _HelloWorld_ application.
-
-====
-----
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  name: gluster-heketi  <1>
-provisioner: kubernetes.io/glusterfs  <2>
- parameters:
-  endpoint: "heketi-storage-endpoints"  <3>
-  resturl: "http://10.42.0.0:8080"  <4>
-  restuser: "joe"  <5>
-  restuserkey: "My Secret Life"  <6>
-----
-<1> Name of the storage class.
-<2> The provisioner.
-<3> The GlusterFS-defined endpoint (`oc get endpoints`).
-<4> Heketi REST URL, taken from Step 1 above (`echo $HEKETI_CLI_SERVER`).
-<5> Rest username. This can be any value since authorization is turned off.
-<6> Rest user key. This can be any value.
-====
-
-. Create the Storage Class YAML file, save it, then submit it to {product-title}:
-+
-----
-$ oc create -f gluster-storage-class.yaml
-storageclass "gluster-heketi" created
-----
-
-. View the storage class:
-+
-----
-$ oc get storageclass
-NAME              TYPE
-gluster-heketi    kubernetes.io/glusterfs
-----
-
-[[create-a-pvc-ro-request-storage-for-your-application]]
-== Create a PVC to Request Storage for Your Application
-
-. Create a persistent volume claim (PVC) requesting 5GB of storage.
-+
-During that time, the Dynamic Provisioning Framework and Heketi will
-automatically provision a new GlusterFS volume and generate the persistent volume
-(PV) object:
-
-====
-----
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
- name: gluster1
-spec:
- accessModes:
-  - ReadWriteOnce
- storageClassName: gluster-heketi  <1>
- resources:
-   requests:
-     storage: 5Gi <2>
-----
-<1> The name of the storage class.
-<2> The amount of storage requested.
+All `oc` commands are executed on the {product-title} master host.
 ====
 
-. Create the PVC YAML file, save it, then submit it to {product-title}:
-+
-----
-$ oc create -f gluster-pvc.yaml
-persistentvolumeclaim "gluster1" created
-----
-
-. View the PVC:
-+
-----
-$ oc get pvc
-NAME       STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
-gluster1   Bound     pvc-7d37c7bd-bb5b-11e6-b81e-525400d87180   5Gi        RWO           14h
-----
-+
-Notice that the PVC is bound to a dynamically created volume.
+[[prerequisites]]
```

Comment:
@jarrpa - this anchor tag is used twice in this PR. Can you change this to be more specific to this topic?